src/views/Drive.vue

```diff
@@ -384,6 +385,20 @@ module.exports = {
 				return false;
 			}
 		},
+		canCreateThumbnail() {
+            if (this.selectedFiles.length != 1)
+                return false;
+            let file = this.selectedFiles[0];
+            if (file.props.thumbnail.ref != null) {
+                return false;
```

Comment:
I can see cases where I might want to recalculate the thumbnail, like when a movie picked a bad frame, or if the thumbnail was made in firefox and I want webp thumbnail (and I'm in chrome)

Reply:
But maybe those aren't important?